test/bitswap-mock-internals.js

```diff
@@ -78,11 +78,11 @@ describe('bitswap with mocks', function () {
             expect(blocks[1].data).to.eql(b2.data)
 
             const ledger = bs.ledgerForPeer(other)
-            expect(ledger.peer).to.equal(other.toB58String())
-            expect(ledger.value).to.equal(0)
-            expect(ledger.sent).to.equal(0)
-            expect(ledger.recv).to.equal(96)
-            expect(ledger.exchanged).to.equal(2)
+            expect(ledger.Peer).to.equal(other.toPrint())
+            expect(ledger.Value).to.equal(0)
+            expect(ledger.Sent).to.equal(0)
+            expect(ledger.Recv).to.equal(96)
+            expect(ledger.Exchanged).to.equal(2)
```

Comment:
We just need to do this change at the http-api level to match go-ipfs. The internals should be doing the JS way.